WcaOnRails/lib/middlewares/warden_user_logger.rb

```diff
@@ -0,0 +1,15 @@
+module Middlewares
+  class WardenUserLogger
+    def initialize(app, logger=Rails.logger)
+      @app = app
+      @logger = logger
+    end
+
+    def call(env)
+      warden_user = env['warden'].user
```

Comment:
Well, it is possible (thanks to Ruby's everything's mutable stance), but not in a clean way. I solved it by mocking `Rails.logger` in 62a159109ea77fd08238ba428ec19386ba61a39e. Well, it works.